### PR TITLE
Hot reload api

### PR DESCRIPTION
**Pull request by Its-Just-Nans**
With this PR, you can now reload easily the entire API (`/services` and `/routes`) whitout restarting the server

To do this :
- be in `development` mode
- do a GET on the URL `.../api/reloadAPI`

If the message is `"ok"` the API has been restarted

### CODE REVIEW

**Comment by Its-Just-Nans** [comment]
Explanation (how it's work) :
We delete router from the `express` stack
We delete cache of require (to reload files)
We re-add router of the api to the `express` stack

File: server.js [hunk 1]
```diff
@@ -121,6 +122,53 @@ const initServer = async () => {
         await middlewarePassport.init(passport);
         require('./services/cache').cacheSetting();
         const apiRouter = require('./routes').InitRoutes(express, server);
+        if (dev) {
+            const reloadRouter  = express.Router();
```
Comment (devaquila) [inline server.js:126]: move with new code to new function

**Review by devaquila — COMMENTED** [review]
move with new code to new function

**Comment by Its-Just-Nans** [comment]
I think the POC can be better if you accept this PR
https://github.com/AquilaCMS/AquilaCMS/pull/124

**Pull request merged**